### ISSUE TITLE
What does “AUTO” mean in matching algorithm?

### ISSUE THREAD

**Issue by graduate0629** [openMVG/openMVG#2323]
Hello! AUTO: auto choice from regions type. Could you please tell me what "AUTO" means in the matching algorithm? Thanks!

**Comment by pmoulon** [comment]
Mean it select the fastest and method that will match your underlying descriptor type https://github.com/openMVG/openMVG/blob/6d6b1dd70bded094ba06024e481dd5a5c662dc83/src/software/SfM/main_ComputeMatches.cpp#L233 aka for scalar vs binary features.

**Comment by graduate0629** [comment]
> Mean it select the fastest and method that will match your underlying descriptor type
> 
> https://github.com/openMVG/openMVG/blob/6d6b1dd70bded094ba06024e481dd5a5c662dc83/src/software/SfM/main_ComputeMatches.cpp#L233
> 
> aka for scalar vs binary features.

Thank you for your prompt reply. As a beginner, I have another question. What does “SfM Scene RMSE” mean in SfM Reconstruction Report? Is this the average projection error?

**Comment by pmoulon** [comment]
Yes it is. Root mean square error. So the average reprojection errors (residuals pixel reprojection error between 3d projected points and 2d features)
